libs/ui/src/screens/tokens/tokens.tsx

```diff
@@ -108,7 +111,6 @@ export const Tokens: FC = () => {
       return (
         <AccountToken
           token={token}
-          loadBalance={!searchValue}
           showButton={showButton}
           theme={TokenItemThemesEnum.Secondary}
           marketInfo={allTokensMarketInfo[tokenMetadataSlug]}
```

Comment:
if token has `dollarBalance`, so you maybe we do not need to send `marketInfo` here